src/@next/Card/CardStyle.ts

```diff
@@ -34,18 +34,38 @@ export const StyledCardHeaderSectionHalf = styled.div`
 `;
 
 export const StyledCardContentWrapper = styled.div`
-  padding-left: ${space4};
+  padding-bottom: ${space4};
```

Comment:
I remove padding-left and right on the wrapper. The reason is, it should't be an issue if we set the root Section to have padding-left-right = 24px, since the sub section padding is overriden